Classes/Handler/Mapping/DefaultMappingHandler.php

```diff
@@ -64,6 +64,7 @@ protected function valueProcessing(array $instructions, array $flexformData, str
                 }
                 break;
             case 'flexform':
+                $GLOBALS['TSFE']->register['tx_templavoilaplus_pi1.current_field'] = $instructions['dataPath'];
```

Comment:
Shouldn't we move this before this switch? It could be also used for "row"? Also add it as function if later more register settings are needed?
The call to registerTypoScriptParentRec method also have a TODO hint, is there a change possible (if yes maybe in another PR).

Reply:
I though only flexform can hold rows, but alas, if you are right, than this helps; and even if I would be right, it won't hurt at all to move earlier.

Concerning the TODO: I don't actually understand it nor the function, so I can't contribute there.